### PR TITLE
Duplicate system.process.cmdline field with process.command_line ECS field name

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## What does this PR do?

This PR is to migrate `system.process.cmdline` field to use `process.command_line` ECS field name instead.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [x] I have made corresponding changes to the documentation
- [ ] ~~I have made corresponding change to the default configuration files~~
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

File: metricbeat/module/system/process/_meta/data.json [hunk 1]
```diff
@@ -11,200 +11,45 @@
...
+        "name": "___TestData_in_",
+        "pgid": 89312,
+        "pid": 89314,
+        "ppid": 89313
     },
     "service": {
         "type": "system"
     },
     "system": {
         "process": {
```
Comment (fearful-symmetry) [inline metricbeat/module/system/process/_meta/data.json:30]: We should probably keep the cgroup events in the data.json.

File: metricbeat/module/system/process/process.go [hunk 2]
```diff
@@ -128,10 +128,11 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) error {
...
 			"process": common.MapStr{
-				"name": getAndRemove(proc, "name"),
-				"pid":  getAndRemove(proc, "pid"),
-				"ppid": getAndRemove(proc, "ppid"),
-				"pgid": getAndRemove(proc, "pgid"),
+				"name":         getAndRemove(proc, "name"),
+				"pid":          getAndRemove(proc, "pid"),
+				"ppid":         getAndRemove(proc, "ppid"),
+				"pgid":         getAndRemove(proc, "pgid"),
+				"command_line": getAndRemove(proc, "cmdline"),
```
Comment (fearful-symmetry) [inline metricbeat/module/system/process/process.go:135]: Do we really want to backport this, or will the `alas` just keep the old value?

**Review by fearful-symmetry — APPROVED** [review]
Lets wait for the tests, otherwise LGTM.

File: metricbeat/module/system/process/process.go (hunk 2 above)
Comment (kaiyan-sheng) [reply metricbeat/module/system/process/process.go]: yep I just updated the PR to duplicate this field instead of replacing it using ECS field.

**Pull request merged**